### ISSUE TITLE
Incomplete guide

### ISSUE THREAD

**Issue by megabytesme** [choss/joycontrol-web-interface#2]
Hi there, I have the rest API setup with joycontroller, however your last step "java -jar" doesn't make sense. I have tried running nearly all of the .jar files in this repo however they all give me the error "Error: Invalid or corrupt jarfile".

**Comment by choss** [comment]
You are right, I should work on making this easier. As there was not much of feedback here, I kind of left it like that, since it worked for me :)

I added as a quick fix some comments to the main readme. I will see that more can follow. At the moment I am not playing AC:NH so much anymore.
